x-pack/plugins/security_solution/public/management/cypress/e2e/mocked_data/artifacts.cy.ts

```diff
@@ -35,7 +35,7 @@ const loginWithoutAccess = (url: string) => {
   loadPage(url);
 };
 
-describe('Artifacts pages', () => {
+describe('Artifacts pages', { tags: '@ess' }, () => {
```

Comment:
just to confirm: `tags` can be `string` or `Array<string>`, right? First instance I'm seeing using string value